src/UI/toolbar/GPSIndicatorPage.qml

```diff
@@ -27,6 +27,25 @@ ToolIndicatorPage {
     property var    rtkSettings:        QGroundControl.settingsManager.rtkSettings
     property bool   useFixedPosition:   rtkSettings.useFixedBasePosition.rawValue
 
+    function errorText() {
+        if (_activeVehicle.gps.systemErrors.value === 1) {
+            return qsTr("Incoming correction")
+        } else if (_activeVehicle.gps.systemErrors.value === 2) {
+            return qsTr("Configuration")
+        } else if (_activeVehicle.gps.systemErrors.value === 4) {
+            return qsTr("Software")
+        } else if (_activeVehicle.gps.systemErrors.value === 8) {
+            return qsTr("Antenna")
+        } else if (_activeVehicle.gps.systemErrors.value === 16) {
+            return qsTr("Event congestion")
+        } else if (_activeVehicle.gps.systemErrors.value === 32) {
+            return qsTr("CPU overload")
+        } else if (_activeVehicle.gps.systemErrors.value === 64) {
+            return qsTr("Output congestion")
+        }
+        return "Multiple errors"
```

Comment:
I was worried that too many errors would make it too crowdy even if it's unlikely to happen, I could try concatenatinge all them as it would make it clearer though